tests/utils/__init__.py

```diff
@@ -8,10 +8,10 @@
 import socket
 import subprocess
 from distutils.spawn import find_executable
-from packaging.version import Version
-from unittest import SkipTest
+from unittest import SkipTest, mock
 
 from hvac import Client
+from packaging.version import Version
```

Comment:
`packaging` is a dep for various packages but should still add it as a dev dep

Reply:
good catch! Added in [54543a4](https://github.com/hvac/hvac/pull/943/commits/54543a41f02cad790530bfbd394016d3e1990b9d).